src/controllers/academicYear.ts

```diff
@@ -0,0 +1,29 @@
+import { Context } from 'koa';
+import { AcademicYearServices } from '../services/academicYear'
+
+export default class AcademicYearControllers {
```

Comment:
Services and Controllers should not be plural. they should be Service and Controller
many line spaces, remove where not needed